### DISCUSSION TITLE
Crop from the bounding box of the cropped Object?

### DISCUSSION THREAD

**Discussion by arrrrny** [danielgatis/rembg#85] [Ideas]
Is there a way to crop image from the bounding box of the cropped object? I am planning to use this tool to remove background it would be great to resize image based on ratio around bounding box  or around bounding box.

**Comment by nikko-guy** [comment]
I would love to see this, or even if someone has a script to add on to this!

**Comment by myanonapps** [comment]
I created a UI around rembg and includes a crop action.  It is fairly easy via Python PIL to crop the image returned or created by rembg

`self.image_pil.crop(self.image_pil.getbbox()`
